psql.go

```diff
@@ -28,15 +28,15 @@ func postgresSetup(db *sql.DB) error {
 	}
 
 	row := db.QueryRow(`SELECT 'DevicesDeviceIDIndex'::regclass`)
-	if err := row.Scan(nil); err != nil {
```

Comment:
This is a style issue. I prefer it with := as the statement is now independent of the surrounding code.